doc/main/reference/task_group_extensions.rst

```diff
@@ -87,6 +89,17 @@ Schedules the task object pointed by the ``h`` for execution.
       * ``h`` is not empty.
       * ``*this`` is the same ``task_group`` that ``h`` is created with.    
 
+.. cpp:function:: task_group_status run_and_wait(task_handle&& h)
+
+The equivalent to ``{run(h); return wait();}``.
+
+.. note::
+   The failure to satisfy the following conditions leads to undefined behavior:
```

Comment:
May be simplify: `If h is not empty or *this is the same task_group that h is created with, the behavior is undefined`?

Reply:
updated by @alexandraepan